{
  TestAssemblies: [
    {
      Name: {SolutionDirectory}test\assets\Json.TestLogger.XUnit.NetFull.Tests\bin\Debug\net46\Json.TestLogger.XUnit.NetFull.Tests.dll,
      Fixtures: [
        {
          Name: UnitTest1,
          Tests: [
            {
              FullyQualifiedName: NUnit.Xml.TestLogger.NetFull.Tests.UnitTest1.Example_Failure,
              Namespace: NUnit.Xml.TestLogger.NetFull.Tests,
              Type: UnitTest1,
              Method: Example_Failure,
              Result: Failed
            },
            {
              FullyQualifiedName: NUnit.Xml.TestLogger.NetFull.Tests.UnitTest1.Example_Success,
              Namespace: NUnit.Xml.TestLogger.NetFull.Tests,
              Type: UnitTest1,
              Method: Example_Success,
              Result: Passed
            }
          ]
        },
        {
          Name: UnitTest2,
          Tests: [
            {
              FullyQualifiedName: NUnit.Xml.TestLogger.Tests2.UnitTest2.Example_Failure,
              Namespace: NUnit.Xml.TestLogger.Tests2,
              Type: UnitTest2,
              Method: Example_Failure,
              Result: Failed
            }
          ]
        }
      ]
    }
  ],
  TestMessages: [
    {
      Message: [xUnit.net _Timestamp_] xUnit.net VSTest Adapter v2.4.3+1b45f5407b (32-bit Desktop .NET 4.0.30319.42000)
    },
    {
      Message: [xUnit.net _Timestamp_]   Discovering: Json.TestLogger.XUnit.NetFull.Tests
    },
    {
      Message: [xUnit.net _Timestamp_]   Discovered:  Json.TestLogger.XUnit.NetFull.Tests
    },
    {
      Message: [xUnit.net _Timestamp_]   Starting:    Json.TestLogger.XUnit.NetFull.Tests
    },
    {
      Level: Error,
      Message: [xUnit.net _Timestamp_]     NUnit.Xml.TestLogger.NetFull.Tests.UnitTest1.Example_Failure [FAIL]
    },
    {
      Message: [xUnit.net _Timestamp_]       Assert.False() Failure
    },
    {
      Message: [xUnit.net _Timestamp_]       Expected: False
    },
    {
      Message: [xUnit.net _Timestamp_]       Actual:   True
    },
    {
      Message: [xUnit.net _Timestamp_]       Stack Trace:
    },
    {
      Message: [xUnit.net _Timestamp_]         {SolutionDirectory}test\assets\Json.TestLogger.XUnit.NetCore.Tests\UnitTest1.cs(18,0): at NUnit.Xml.TestLogger.NetFull.Tests.UnitTest1.Example_Failure()
    },
    {
      Level: Error,
      Message: [xUnit.net _Timestamp_]     NUnit.Xml.TestLogger.Tests2.UnitTest2.Example_Failure [FAIL]
    },
    {
      Message: [xUnit.net _Timestamp_]       Assert.False() Failure
    },
    {
      Message: [xUnit.net _Timestamp_]       Expected: False
    },
    {
      Message: [xUnit.net _Timestamp_]       Actual:   True
    },
    {
      Message: [xUnit.net _Timestamp_]       Stack Trace:
    },
    {
      Message: [xUnit.net _Timestamp_]         {SolutionDirectory}test\assets\Json.TestLogger.XUnit.NetCore.Tests\UnitTest2.cs(12,0): at NUnit.Xml.TestLogger.Tests2.UnitTest2.Example_Failure()
    },
    {
      Message: [xUnit.net _Timestamp_]   Finished:    Json.TestLogger.XUnit.NetFull.Tests
    }
  ]
}